packages/kit/src/__tests__/compute-limit-internal-test.ts

```diff
@@ -168,47 +170,51 @@ describe('getComputeUnitEstimateForTransactionMessage_INTERNAL_ONLY_DO_NOT_EXPOR
-                                role: AccountRole.WRITABLE,
-                            },
-                            {
-                                address: 'SysvarRecentB1ockHashes11111111111111111111' as Address,
-                                role: AccountRole.READONLY,
-                            },
-                            {
-                                address: 'HzMoc78z1VNNf9nwD4Czt6CDYEb9LVD8KsVGP46FEmyJ' as Address,
-                                role: AccountRole.READONLY_SIGNER,
-                            },
-                        ],
-                        data: new Uint8Array([4, 0, 0, 0]),
-                        programAddress: '11111111111111111111111111111111' as Address,
-                    },
-                ],
-                lifetimeConstraint: {
-                    nonce: 'BzAqD6382v5r1pcELoi8HWrBDV4dSL9NGemMn2JYAhxc' as Nonce,
+        const transactionMessage = {
+            ...mockTransactionMessage,
+            instructions: [
+                {
+                    accounts: [
+                        {
+                            address: '7wJFRFuAE9x5Ptnz2VoBWsfecTCfuuM2sQCpECGypnTU' as Address,
+                            role: AccountRole.WRITABLE,
+                        },
+                        {
+                            address: 'SysvarRecentB1ockHashes11111111111111111111' as Address,
+                            role: AccountRole.READONLY,
+                        },
+                        {
+                            address: 'HzMoc78z1VNNf9nwD4Czt6CDYEb9LVD8KsVGP46FEmyJ' as Address,
+                            role: AccountRole.READONLY_SIGNER,
+                        },
+                    ],
+                    data: new Uint8Array([4, 0, 0, 0]),
+                    programAddress: '11111111111111111111111111111111' as Address,
                 },
+            ],
+            lifetimeConstraint: {
+                nonce: 'BzAqD6382v5r1pcELoi8HWrBDV4dSL9NGemMn2JYAhxc' as Nonce,
             },
+        };
+
+        getComputeUnitEstimateForTransactionMessage_INTERNAL_ONLY_DO_NOT_EXPORT({
+            rpc,
+            transactionMessage,
         }).catch(() => {});
         expect(simulateTransaction).toHaveBeenCalledWith(
             expect.anything(),
             expect.objectContaining({ replaceRecentBlockhash: false }),
         );
     });
     it('asks for a replacement blockhash even when the transaction message has a blockhash lifetime', () => {
+        const transactionMessage = {
+            ...mockTransactionMessage,
+            lifetimeConstraint: MOCK_BLOCKHASH_LIFETIME_CONSTRAINT,
+        };
+
         getComputeUnitEstimateForTransactionMessage_INTERNAL_ONLY_DO_NOT_EXPORT({
             rpc,
-            transactionMessage: {
-                ...mockTransactionMessage,
-                lifetimeConstraint: MOCK_BLOCKHASH_LIFETIME_CONSTRAINT,
-            },
+            transactionMessage,
```

Comment:
Per #581, this is one of those opportunities where we can just leave the `lifetimeConstraint` out rather than to supply a fake one right? ie. this should be split into two tests, one where you don't supply one at all, and one where you supply one that just gets written over.